cmd/security-secretstore-setup/res/configuration.yaml

```diff
@@ -0,0 +1,84 @@
+#################################################################################
+# Copyright 2019 Dell Inc.
+# Copyright 2020-2023 Intel Corp.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+#
+#################################################################################
+
+LogLevel: DEBUG
+SecretStore:
+  Type: vault
+  Protocol: http
+  Host: localhost
+  Port: 8200
+  CertPath: ""
+  CaFilePath: ""
+  CertFilePath: ""
+  KeyFilePath: ""
+  TokenFolderPath: /vault/config/assets
+  TokenFile: resp-init.json
+  VaultSecretShares: 5
+  VaultSecretThreshold: 3
+  TokenProvider: /security-file-token-provider
+  TokenProviderArgs:
+    - "-configDir"
```

Comment:
interesting, is it because it starts with - for configDir so we need to put it into double quotes?

Reply:
Yep, seems so